src/options/options.ts

```diff
@@ -176,6 +176,17 @@ $( document ).ready( async () => {
   $( '#passphraseBox' ).keydown( entryModalOnEnter );
   $( '#baseBox' ).keydown( entryModalOnEnter );
   $( '#tagsBox' ).keydown( entryModalOnEnter );
+
+  const presetDropdownItems = $( '#presetDropdownItems' );
```

Comment:
You can type this as an `HTMLDivElement`

Reply:
I don't know if `HTMLDivElement` will work with jQuery `.click()`.

Reply:
fucking jquery...